node/resolvers/providerLogo.ts

```diff
@@ -0,0 +1,16 @@
+import { Image } from 'vtex.geolocation-graphql-interface'
+
+// must be the same as assets/logo.png
+const SRC_URL =
```

Comment:
This URL was generated by uploading the Google Logo to a Github commentary and grabbing the link. The reason why I did this (and pushed...) is to set up a discussion on _where_ to host the image - I can think of few places:

1. Github: the same image is on `node/assets/logo.png`, maybe we can just grab the link from there, but I get that we would depend on Github, which kinda sucks
2. VTEX (file manager): we can upload this using the [file-manager API](https://github.com/vtex/file-manager), but it gets restricted by Account and App, which seems weird to me. Also, I'm not sure if everyone would have access to it.
3. VTEX (?): just host somewhere in our infra, in a way that the content is public and unique.
4. Leave it as is and monitor it to fix when something bad happens :-)

Also, I don't think this should be generated in real time or per account/app, it would just increase complexity and add unnecessary copies of the image all around.

Reply:
> we should host this on a VTEX server, can you upload this using the assets builder?

@lucasecdb I cannot retrieve the URL assets builder generates :/

Reply:
Solution on [`ee5f224` (#4)](https://github.com/vtex-apps/google-geolocation-resolver/pull/4/commits/ee5f2246cd1d8d44d9f8c50cde8e80e630f75cea)